### PR TITLE
[MIG] base_locale_uom_default: Migration to 12.0

### PR DESCRIPTION
**Pull request by skukered**

### CODE REVIEW

File: base_locale_uom_default/__manifest__.py [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+# Copyright 2019 Rubén Bravo <rubenred18@gmail.com>
+# License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl).
+
+{
+    'name': 'Locale - Default UoM',
+    'summary': 'This provides settings to select default UoMs at the '
+               'language level.',
+    'version': '12.0.1.0.0',
+    'category': 'Extra Tools',
+    'website': 'https://laslabs.com/',
```
Comment (tbaden) [inline base_locale_uom_default/__manifest__.py]: please set https://github.com/OCA/server-tools/ as website

File: base_locale_uom_default/__init__.py [hunk 2]
```diff
@@ -0,0 +1,5 @@
+# -*- coding: utf-8 -*-
```
Comment (tbaden) [inline base_locale_uom_default/__init__.py]: please remove this line. no longer needed

File: base_locale_uom_default/__manifest__.py [hunk 3]
```diff
@@ -0,0 +1,27 @@
+# -*- coding: utf-8 -*-
```
Comment (tbaden) [inline base_locale_uom_default/__manifest__.py]: please remove this line. no longer needed

File: base_locale_uom_default/models/__init__.py [hunk 4]
```diff
@@ -0,0 +1,5 @@
+# -*- coding: utf-8 -*-
```
Comment (tbaden) [inline base_locale_uom_default/models/__init__.py]: please remove this line. no longer needed

File: base_locale_uom_default/models/res_lang.py [hunk 5]
```diff
@@ -0,0 +1,56 @@
+# -*- coding: utf-8 -*-
```
Comment (tbaden) [inline base_locale_uom_default/models/res_lang.py]: please remove this line. no longer needed

File: base_locale_uom_default/tests/__init__.py [hunk 6]
```diff
@@ -0,0 +1,5 @@
+# -*- coding: utf-8 -*-
```
Comment (tbaden) [inline base_locale_uom_default/tests/__init__.py]: please remove this line. no longer needed

File: base_locale_uom_default/tests/test_res_lang.py [hunk 7]
```diff
@@ -0,0 +1,45 @@
+# -*- coding: utf-8 -*-
```
Comment (tbaden) [inline base_locale_uom_default/tests/test_res_lang.py]: please remove this line. no longer needed

**Review by tbaden — CHANGES_REQUESTED** [review]
thanks for the work 👍 
I did a code review:
some changes are required.
please use the fragment [readme](https://github.com/OCA/maintainer-tools/wiki/Migration-to-version-12.0).

**Comment by skukered** [comment]
@tbaden changes done.

File: base_locale_uom_default/tests/__init__.py (hunk 6 above)
Comment (tbaden) [reply base_locale_uom_default/tests/__init__.py]: you missed that one @skukered

Comment (skukered) [reply base_locale_uom_default/tests/__init__.py]: Sorry... Done

**Review by tbaden — APPROVED** [review]

**Review by benbrich — APPROVED** [review]
I have just just stumbled over this one and did a functional check. Everything seems to work like it should. Thumbs up & thanks for the work. Let's get this pr done

**Pull request merged**